deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1266,7 +1266,8 @@ class ChatViewController: UITableViewController {
         } else {
             // new outgoing message
```

Comment:
i think, this else-branch is not only about "new outgoing message" but also about incoming messages, not in the chatlist for whatever reason, so, it still may be wrong to insert that message to the end.

however, this is another issue, so, we can leave that as is for now.